tailwind.base.css

```diff
@@ -135,10 +131,26 @@
     @apply flex items-center justify-center px-4 py-1 text-white rounded backdrop-blur;
   }
 
+  /* GENERIC TEXT INPUT (textarea, text field, url input...) */
+  .gn-ui-text-input {
+    --rounded: var(--gn-ui-text-input-rounded, 0.25em);
+    --padding: var(--gn-ui-text-input-padding, 0.6em);
+    @apply appearance-none border-2 p-[--padding] rounded-[--rounded] w-full transition-colors duration-75
```

Comment:
That's very nice and is so much nicer that the previous way of doing the base css class string !! I really need to take a deeper look at `@apply`.